Source/WebKit/UIProcess/API/Cocoa/WKNavigationResponsePrivate.h

```diff
@@ -31,5 +31,7 @@
 @property (nonatomic, readonly) NSURLRequest *_request;
 @property (nonatomic, readonly) NSString *_downloadAttribute WK_API_AVAILABLE(macos(10.15), ios(13.0));
 @property (nonatomic, readonly) BOOL _wasPrivateRelayed WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
+@property (nonatomic, readonly) NSString *_proxyName WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
I think we might want this to be nullable also.  Then we can use nsStringNilIfNull in the accessor, and it'll be clear when there is no proxy.